test/fixtures/testFiles/invalidLinkRoutingTest.md

```diff
@@ -2,8 +2,8 @@ This invalid link to a file with an anchor is routed to a destination where the
 This invalid link to a file is routed to a destination where the file doesn't exist [invalidLink](../dir/invalidLink.md)
 This invalid link to a html file with an anchor that needs to be resolved as markdown is routed to a destination where the file doesn't exist [invalidAnchorLink](../dir/invalidLink.html#does-not-exist)
 This invalid link to html file that needs to be resolved as markdown is routed to a destination where the file doesn't exist [invalidLink](../dir/invalidLink.html)
-This invalid link to a file with an anchor is routed to a valid destination, but the anchor is non-existent [invalidAnchorLink](../../subdir/linkTestFile.md#header-7)
-This invalid link to a file has no matched routing [invalidAnchorLink](../../dir/subdir/linkTestFile.md)
-This invalid link to a file with an anchor has no matched routing [invalidAnchorLink](../../dir/subdir/linkTestFile.md#header-7)
-This invalid link to a file is routed to an invalid destination [invalidAnchorLink](../dir/subdir/linkTestFile.md)
+This invalid link to a file with an anchor is routed to a valid destination, but the anchor is non-existent [invalidAnchorLink](../../subDir/linkTestFile.md#header-7)
+This invalid link to a file has no matched routing [invalidAnchorLink](../../dir/subDir/linkTestFile.md)
+This invalid link to a file with an anchor has no matched routing [invalidAnchorLink](../../dir/subDir/linkTestFile.md#header-7)
+This invalid link to a file is routed to an invalid destination [invalidAnchorLink](../dir/subDir/linkTestFile.md)
```

Comment:
Why did these need to change?

Reply:
Needed to change this to match the capitalization of the directory name so that unit tests will pass on ubuntu, it only worked locally before because macos is case insensitive for paths.

Reply:
I didn't realize they originally didn't match. Thanks for explaining.